src/SFA.DAS.Data.Functions/Ioc/StructureMapObjectResolver.cs

```diff
@@ -1,27 +1,22 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using StructureMap;
 
 namespace SFA.DAS.Data.Functions.Ioc
 {
     public class StructureMapObjectResolver : IObjectResolver
     {
         private static IContainer _container;
-        private static Object _lockObject;
+        private static readonly object LockObject = new object();
```

Comment:
class naming convention is not consistant, should this be _lockObject as per original?